src/main/java/frc/robot/RobotContainer.java

```diff
@@ -41,7 +43,7 @@ public class RobotContainer {
 
   private final Drivetrain subDrivetrain = new Drivetrain();
   private final Arm subArm = new Arm();
-  private final Intake subIntake = new Intake();
+  private final Intake subIntake = new Intake(conDriver, conOperator);
```

Comment:
the intake subsystem shouldn't be what uses the controllers, there should be a command similar to the leds command to control the controller rumbling